EzMap.Domain/Models/PoiConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace EzMap.Domain;
+
+internal class PoiConfiguration :  IEntityTypeConfiguration<Poi>
+{
+    public void Configure(EntityTypeBuilder<Poi> builder)
+    {
+        builder.ToTable("Poi").HasKey(p => p.PoiId);
```

Comment:
No hard coded string

Reply:
hmm, do I need to create an enum?

Reply:
See this comment https://github.com/Ez-Map/ezmap-poc/pull/1/files#r1337532625